cmd/apiserver-proxy-sidecar/main.go

```diff
@@ -32,6 +32,7 @@ func parseAndValidateFlags() *app.ConfigParams {
 		"[optional] indicates if the sidecar should run as a daemon")
 	flag.StringVar(&params.IPAddress, "ip-address", "", "ip-address on which the proxy is listening (e.g. 1.2.3.4).")
 	flag.StringVar(&params.LocalPort, "port", "9443", "[optional] port on which the proxy is listening.")
+	flag.BoolVar(&params.ManageInterface, "manage-interface", false, "[optional] indicates wether the interface should be created/deleted by the app")
```

Comment:
Why add another command line option? If you look way back in the `apiserver-proxy` history it used to create the device if it did not exist. Would that be an option now as well. In other words, why not simply use the device name specified via `--interface` and if it does not exist create it?

(You can also see the history in the documentation of `--sync-interval` and `--cleanup`.)